monai/transforms/utils.py

```diff
@@ -392,7 +453,59 @@ def _correct_centers(
         center = np.unravel_index(indices_to_use[random_int], label_spatial_shape)
         # shift center to range of valid centers
         center_ori = list(center)
-        centers.append(_correct_centers(center_ori, valid_start, valid_end))
+        centers.append(correct_crop_centers(center_ori, spatial_size, label_spatial_shape))
+
+    return centers
+
+
+def generate_label_classes_crop_centers(
+    spatial_size: Union[Sequence[int], int],
+    num_samples: int,
+    ratios: List[Union[float, int]],
```

Comment:
might be useful to set `ratios` to `ones(len(indices))` by default?

Reply:
Sure, added default values in the latest commit.

Thanks.